### PR TITLE
Updating Authentication Redesign

### PR DESCRIPTION
**Pull request by cmckee-dev**
### Enhancement / Fix

This PR updates styles for authentication pages for users who aren't logged in. It removes the site header and footer to give us a middle ground between public facing and our web app styles. 

This also removes Omni-auth references from the application. CVE-2015-9284

### General

- [x] My code follows the style guidelines of this project & the tests are passing
- [x] I have performed a self-review of my own code
- [x] I have updated DB schema annotations or I have not made any DB schema changes

### CodeFund Core Team

- [x] I have added the appropriate milestone if applicable
- [x] I have requested a review from another developer(s)

## Screenshots

Every authentication page looks the same as previously minus removing the header and footer.

Screenshot_2019-11-19 Sessions

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -27,16 +27,9 @@ class Application < Rails::Application
...
-      [Devise::SessionsController,
-       Devise::RegistrationsController,
-       Devise::ConfirmationsController,
-       Devise::UnlocksController,
-       Devise::InvitationsController,
-       Devise::PasswordsController,
-       UserPasswordsController,].each do |views|
-        views.layout proc { |controller| Rails.env.development? && ENV["REDESIGN"] == "true" ? "application_redesign" : "authentication" }
+      if Rails.env.development? && ENV["REDESIGN"]
+        ApplicationController.layout "application_redesign"
```
Comment (hopsoft) [inline config/application.rb]: Nice.

File: app/views_redesigned/devise/passwords/new.html.erb [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<%= render "devise/layout" do %>
+<%# <%= render "devise/layout" do %1> %>
```
Comment (andrewmcodes) [inline app/views_redesigned/devise/passwords/new.html.erb]: If we aren't using this, go ahead and remove the commented out code.

File: app/models/user.rb [hunk 3]
```diff
@@ -153,13 +153,11 @@ class User < ApplicationRecord
...
     :invitable,
     :lockable,
-    :omniauthable,
     :recoverable,
     :rememberable,
     :timeoutable,
     :trackable,
-    :validatable,
-    omniauth_providers: [:github]
+    :validatable
```
Comment (andrewmcodes) [inline app/models/user.rb]: I believe we should have a trailing comma here

Comment (cmckee-dev) [reply app/models/user.rb]: `./bin/standardize` yells at us for this.

File: config/application.rb (hunk 1 above)
Comment (cmckee-dev) [reply config/application.rb]: I think this is a good move for when we move the redesign to stage/prod. It allows us to easily remove the excess code.

File: app/views_redesigned/devise/passwords/new.html.erb (hunk 2 above)
Comment (cmckee-dev) [reply app/views_redesigned/devise/passwords/new.html.erb]: Nice :+1:

**Pull request closed without merging**